core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -465,8 +465,7 @@ public final boolean containsObject(IN_NAME name, Object value) {
 
     @Override
     public final boolean containsBoolean(IN_NAME name, boolean value) {
-        final Boolean v = getBoolean(name);
-        return v != null && v == value;
+         return contains(name, String.valueOf(value)) || contains(name, value ? "1" : "0");
```

Comment:
FYI) I'm now checking HttpHeader related codes to solve https://github.com/line/armeria/issues/5146 issue.

And I found that only `containsBoolean` use `getBoolean` unlike other `contains` 🤔

> https://github.com/line/armeria/pull/3600/files#diff-04ab698cc366a8f18275f83a073bd5c6cc9bf5af0623ef137ee5a716065c2443R467-R472

Maybe just copy&paste `getBoolean` code on `containsBoolean` by mistake on above PR?

Reply:
https://github.com/line/armeria/blob/a8a9ca2f3baf3f001b73a2decbcc9c38e3da2e23/core/src/main/java/com/linecorp/armeria/internal/common/util/StringUtil.java#L29-L35

FYI), `getBoolean` supports `"true", "false", "1", "0"` so I think we should check `"1", "0"` on containsBoolean too~!